src/main/resources/application.properties

```diff
@@ -1 +1,19 @@
+server.port=8080
+spring.jpa.database=POSTGRESQL
+spring.datasource.platform=postgres
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.username=postgres
+spring.datasource.password=gronostaj
+spring.jpa.show-sql=true
+spring.jpa.generate-ddl=true
+spring.jpa.hibernate.ddl-auto=none
```

Comment:
Don't we want to put **create** here and create fresh db schema every app execution?